server/src/logic/controllers/conventionController.js

```diff
@@ -63,46 +39,10 @@ class ConventionController {
       info_qualiopi_info: info_datagouv_ofs.qualiopi ? "OUI" : "NON",
       info_datagouv_ofs_info: info_datagouv_ofs.info,
       nda: info_datagouv_ofs.data?.numerodeclarationactivite || null,
-      ...conventionnementInfos,
+      catalogue_published: (info_datagouv_ofs.qualiopi ? "OUI" : "NON") === "OUI",
```

Comment:
On peut peut-être simplifier cette ligne ?
```suggestion
      catalogue_published: info_datagouv_ofs.qualiopi,
```

Reply:
J'ai préféré conservé le check, car je ne sais pas si info_datagouv_ofs.qualiopi est un boolean à true / false ou si c'est truthy

Reply:
Ok, je te confirme que c'est un booléen (dans `conventionController#findInfoDataGouv`) :
````
    qualiopi:
          result.numerodeclarationactivite && result.certifications_actionsdeformationparapprentissage === "true",
```

Reply:
Tes suggestions ont été appliquées